frontend/src/features/mapSideBar/SidebarContents/subforms/LandSearchForm.tsx

```diff
@@ -0,0 +1,163 @@
+import SearchButton from 'components/common/form/SearchButton';
+import React, { useState } from 'react';
+import { Col, Form, Row } from 'react-bootstrap';
+import { Label } from 'components/common/Label';
+import { Input, FastInput } from 'components/common/form';
+import { pidFormatter } from 'features/properties/components/forms/subforms/PidPinForm';
+import { withNameSpace } from 'utils/formUtils';
+import { GeocoderAutoComplete } from 'features/properties/components/GeocoderAutoComplete';
+import { getIn, useFormikContext } from 'formik';
+import ClickAwayListener from 'react-click-away-listener';
+import { IGeocoderResponse } from 'hooks/useApi';
+import { IParcel } from 'actions/parcelsActions';
+import { ISteppedFormValues } from 'components/common/form/StepForm';
+import { ISearchFields } from '../LandForm';
+import { ReactComponent as ParcelDraftIcon } from 'assets/images/draft-parcel-icon.svg';
+import styled from 'styled-components';
+
+const SearchMarkerButton = styled.button`
+  top: 20px;
+  right: 20px;
+  border: 0px;
+  background-color: none;
+  display: flex;
+`;
+
+interface ISearchFormProps {
+  /** used for determining nameSpace of field */
+  nameSpace?: string;
+  /** handle the population of Geocoder information */
+  handleGeocoderChanges: (data: IGeocoderResponse, nameSpace?: string) => Promise<void>;
+  /** help set the cursor type when click the add marker button */
+  setMovingPinNameSpace: (nameSpace?: string) => void;
+  /** handle the pid formatting on change */
+  handlePidChange: (pid: string, nameSpace?: string) => void;
+  /** handle the pin formatting on change */
+  handlePinChange: (pin: string, nameSpace?: string) => void;
+}
+
+/**
+ * Search component which displays a vertically stacked set of search fields, used to find matching parcels within pims or the parcel layer.
+ * @param {ISearchFormProps} param0
+ */
+const LandSearchForm = ({
```

Comment:
This used to be an embedded component. Moving it to its own file to increase readability.